src/containers/Chat/ChatMessages/ChatMessages.tsx

```diff
@@ -328,10 +330,17 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({ contactId }) => {
     <Container className={styles.ChatMessages} disableGutters>
       {dialogBox}
       {toastMessage}
-      <ContactBar contactName={conversations.contact.name} />
-      <Container className={styles.MessageList} data-testid="messageContainer">
-        {messageList}
-      </Container>
+      <ContactBar contactName={conversations ? conversations.contact.name : 'No contacts'} />
+      {/* To avoid warnings with empty children in Container */}
+      {messageList ? (
+        <Container className={styles.MessageList} data-testid="messageContainer">
+          {messageList}
+        </Container>
+      ) : (
+        <div className={styles.MessageList} data-testid="messageContainer">
+          {messageList}
```

Comment:
I think the Container & div pairing is fine. Just one question, when there's nothing in the message list, shouldn't we be showing an appropriate message to the user. Like `No messages for this conversation` or something?

Reply:
Added this message.